src/tests/assetAmountInput.tsx

```diff
@@ -0,0 +1,13 @@
+import { formatAssetAmountInput } from '../utils/common';
+
+describe('formatAmount', () => {
+  it('should format transaction amount to replace leading dot with 0.', () => {
+    expect(formatAssetAmountInput('.')).toBe('0.');
+  });
+  it('should format transaction amount to replace leading dot with 0.', () => {
+    expect(formatAssetAmountInput('0.1')).toBe('0.1');
+  });
+  it('should format transaction amount to replace leading dot with 0.', () => {
+    expect(formatAssetAmountInput('1')).toBe('1');
+  });
```

Comment:
Your test case descriptions are currently all the same, could you differentiate them please?